Docker images pull request

NataliiaRemezova opened the pull request:
Test pull request of docker-images and workflows into main

justingebert commented:
Terraform Plan Output:
```Running plan in Terraform Cloud. Output will stream here. Pressing Ctrl-C
will stop streaming the logs, but will not stop the plan running remotely.

Preparing the remote plan...

To view this run in a browser, visit:
https://app.terraform.io/app/hypnobuddy/hypnobuddy/runs/run-d2RC1WrpYWh3LqXR

Waiting for the plan to start...

Terraform v1.7.1
on linux_amd64
Initializing plugins and modules...
data.google_secret_manager_secret_version.mongo_url: Refreshing...
data.google_secret_manager_secret_version.mongo_url: Refresh complete after 0s [id=projects/897359304801/secrets/mongo_url/versions/1]
google_cloud_run_v2_service.backend: Refreshing state... [id=projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend]
google_cloud_run_service_iam_binding.default: Refreshing state... [id=v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker]

No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration
and found no differences, so no changes are needed.

─────────────────────────────────────────────────────────────────────────────

Note: You didn't use the -out option to save this plan, so Terraform can't
guarantee to take exactly these actions if you run "terraform apply" now.
```

justingebert commented:
Terraform Plan Output:
```Running plan in Terraform Cloud. Output will stream here. Pressing Ctrl-C
will stop streaming the logs, but will not stop the plan running remotely.

Preparing the remote plan...

To view this run in a browser, visit:
https://app.terraform.io/app/hypnobuddy/hypnobuddy/runs/run-VGKMRVhR4ga99YGv

Waiting for the plan to start...

Terraform v1.7.1
on linux_amd64
Initializing plugins and modules...
data.google_secret_manager_secret_version.mongo_url: Refreshing...
data.google_secret_manager_secret_version.mongo_url: Refresh complete after 0s [id=projects/897359304801/secrets/mongo_url/versions/1]
google_cloud_run_v2_service.backend: Refreshing state... [id=projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend]
google_cloud_run_service_iam_binding.default: Refreshing state... [id=v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker]

No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration
and found no differences, so no changes are needed.

─────────────────────────────────────────────────────────────────────────────

Note: You didn't use the -out option to save this plan, so Terraform can't
guarantee to take exactly these actions if you run "terraform apply" now.
```

NataliiaRemezova commented:
@justingebert, added my workflows and docker images. I renamed all docker files and workflows, so that they don't affect the deployment and can be used for development purposes. There are also no merge conflicts. Only the test pipeline is failing because of the issue with one goalController test. Could you please review and approve the pull request?

justingebert commented:
Terraform Plan Output:
```Running plan in Terraform Cloud. Output will stream here. Pressing Ctrl-C
will stop streaming the logs, but will not stop the plan running remotely.

Preparing the remote plan...

To view this run in a browser, visit:
https://app.terraform.io/app/hypnobuddy/hypnobuddy/runs/run-NLyNabtiXqi7qekk

Waiting for the plan to start...

Terraform v1.7.1
on linux_amd64
Initializing plugins and modules...
data.google_secret_manager_secret_version.mongo_url: Refreshing...
data.google_secret_manager_secret_version.mongo_url: Refresh complete after 0s [id=projects/897359304801/secrets/mongo_url/versions/1]
google_cloud_run_v2_service.backend: Refreshing state... [id=projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend]
google_cloud_run_service_iam_binding.default: Refreshing state... [id=v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker]
google_cloud_run_service_iam_binding.default: Drift detected (update)
google_cloud_run_v2_service.backend: Drift detected (delete)

Note: Objects have changed outside of Terraform

Terraform detected the following changes made outside of Terraform since the
last "terraform apply" which may have affected this plan:

  # google_cloud_run_v2_service.backend has been deleted
  - resource "google_cloud_run_v2_service" "backend" {
        id                      = "projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend"
      - location                = "europe-west1" -> null
      - name                    = "hypnobuddy-backend" -> null
        # (24 unchanged attributes hidden)

        # (2 unchanged blocks hidden)
    }


Unless you have made equivalent changes to your configuration, or ignored the
relevant attributes using ignore_changes, the following plan may include
actions to undo or respond to these changes.

─────────────────────────────────────────────────────────────────────────────

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create
  ~ update in-place

Terraform will perform the following actions:

  # google_cloud_run_service_iam_binding.default will be updated in-place
  ~ resource "google_cloud_run_service_iam_binding" "default" {
        id       = "v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker"
      ~ members  = [
          + "allUsers",
        ]
        # (5 unchanged attributes hidden)
    }

  # google_cloud_run_v2_service.backend will be created
  + resource "google_cloud_run_v2_service" "backend" {
      + conditions              = (known after apply)
      + create_time             = (known after apply)
      + creator                 = (known after apply)
      + delete_time             = (known after apply)
      + effective_annotations   = (known after apply)
      + effective_labels        = (known after apply)
      + etag                    = (known after apply)
      + expire_time             = (known after apply)
      + generation              = (known after apply)
      + id                      = (known after apply)
      + ingress                 = (known after apply)
      + last_modifier           = (known after apply)
      + latest_created_revision = (known after apply)
      + latest_ready_revision   = (known after apply)
      + launch_stage            = (known after apply)
      + location                = "europe-west1"
      + name                    = "hypnobuddy-backend"
      + observed_generation     = (known after apply)
      + project                 = "imi-b4"
      + reconciling             = (known after apply)
      + terminal_condition      = (known after apply)
      + terraform_labels        = (known after apply)
      + traffic_statuses        = (known after apply)
      + uid                     = (known after apply)
      + update_time             = (known after apply)
      + uri                     = (known after apply)

      + template {
          + max_instance_request_concurrency = (known after apply)
          + service_account                  = (known after apply)
          + timeout                          = (known after apply)

          + containers {
              + image = "justingebert/hypnobuddy-backend:latest"

              + env {
                  + name  = "MONGO_URL"
                  + value = (sensitive value)
                }
            }
        }
    }

Plan: 1 to add, 1 to change, 0 to destroy.

─────────────────────────────────────────────────────────────────────────────

Note: You didn't use the -out option to save this plan, so Terraform can't
guarantee to take exactly these actions if you run "terraform apply" now.
```

justingebert commented:
Terraform Plan Output:
```Running plan in Terraform Cloud. Output will stream here. Pressing Ctrl-C
will stop streaming the logs, but will not stop the plan running remotely.

Preparing the remote plan...

To view this run in a browser, visit:
https://app.terraform.io/app/hypnobuddy/hypnobuddy/runs/run-feuVffURkpNAEspG

Waiting for the plan to start...

Terraform v1.7.1
on linux_amd64
Initializing plugins and modules...
data.google_secret_manager_secret_version.mongo_url: Refreshing...
data.google_secret_manager_secret_version.mongo_url: Refresh complete after 0s [id=projects/897359304801/secrets/mongo_url/versions/1]
google_cloud_run_v2_service.backend: Refreshing state... [id=projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend]
google_cloud_run_service_iam_binding.default: Refreshing state... [id=v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker]
google_cloud_run_service_iam_binding.default: Drift detected (update)
google_cloud_run_v2_service.backend: Drift detected (delete)

Note: Objects have changed outside of Terraform

Terraform detected the following changes made outside of Terraform since the
last "terraform apply" which may have affected this plan:

  # google_cloud_run_v2_service.backend has been deleted
  - resource "google_cloud_run_v2_service" "backend" {
        id                      = "projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend"
      - location                = "europe-west1" -> null
      - name                    = "hypnobuddy-backend" -> null
        # (24 unchanged attributes hidden)

        # (2 unchanged blocks hidden)
    }


Unless you have made equivalent changes to your configuration, or ignored the
relevant attributes using ignore_changes, the following plan may include
actions to undo or respond to these changes.

─────────────────────────────────────────────────────────────────────────────

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create
  ~ update in-place

Terraform will perform the following actions:

  # google_cloud_run_service_iam_binding.default will be updated in-place
  ~ resource "google_cloud_run_service_iam_binding" "default" {
        id       = "v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker"
      ~ members  = [
          + "allUsers",
        ]
        # (5 unchanged attributes hidden)
    }

  # google_cloud_run_v2_service.backend will be created
  + resource "google_cloud_run_v2_service" "backend" {
      + conditions              = (known after apply)
      + create_time             = (known after apply)
      + creator                 = (known after apply)
      + delete_time             = (known after apply)
      + effective_annotations   = (known after apply)
      + effective_labels        = (known after apply)
      + etag                    = (known after apply)
      + expire_time             = (known after apply)
      + generation              = (known after apply)
      + id                      = (known after apply)
      + ingress                 = (known after apply)
      + last_modifier           = (known after apply)
      + latest_created_revision = (known after apply)
      + latest_ready_revision   = (known after apply)
      + launch_stage            = (known after apply)
      + location                = "europe-west1"
      + name                    = "hypnobuddy-backend"
      + observed_generation     = (known after apply)
      + project                 = "imi-b4"
      + reconciling             = (known after apply)
      + terminal_condition      = (known after apply)
      + terraform_labels        = (known after apply)
      + traffic_statuses        = (known after apply)
      + uid                     = (known after apply)
      + update_time             = (known after apply)
      + uri                     = (known after apply)

      + template {
          + max_instance_request_concurrency = (known after apply)
          + service_account                  = (known after apply)
          + timeout                          = (known after apply)

          + containers {
              + image = "justingebert/hypnobuddy-backend:latest"

              + env {
                  + name  = "MONGO_URL"
                  + value = (sensitive value)
                }
            }
        }
    }

Plan: 1 to add, 1 to change, 0 to destroy.

─────────────────────────────────────────────────────────────────────────────

Note: You didn't use the -out option to save this plan, so Terraform can't
guarantee to take exactly these actions if you run "terraform apply" now.
```

justingebert commented:
Terraform Plan Output:
```Running plan in Terraform Cloud. Output will stream here. Pressing Ctrl-C
will stop streaming the logs, but will not stop the plan running remotely.

Preparing the remote plan...

To view this run in a browser, visit:
https://app.terraform.io/app/hypnobuddy/hypnobuddy/runs/run-tqMA37FmHCziWqFu

Waiting for the plan to start...

Terraform v1.7.1
on linux_amd64
Initializing plugins and modules...
data.google_secret_manager_secret_version.mongo_url: Refreshing...
data.google_secret_manager_secret_version.mongo_url: Refresh complete after 1s [id=projects/897359304801/secrets/mongo_url/versions/1]
google_cloud_run_v2_service.backend: Refreshing state... [id=projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend]
google_cloud_run_service_iam_binding.default: Refreshing state... [id=v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker]
google_cloud_run_service_iam_binding.default: Drift detected (update)
google_cloud_run_v2_service.backend: Drift detected (delete)

Note: Objects have changed outside of Terraform

Terraform detected the following changes made outside of Terraform since the
last "terraform apply" which may have affected this plan:

  # google_cloud_run_v2_service.backend has been deleted
  - resource "google_cloud_run_v2_service" "backend" {
        id                      = "projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend"
      - location                = "europe-west1" -> null
      - name                    = "hypnobuddy-backend" -> null
        # (24 unchanged attributes hidden)

        # (2 unchanged blocks hidden)
    }


Unless you have made equivalent changes to your configuration, or ignored the
relevant attributes using ignore_changes, the following plan may include
actions to undo or respond to these changes.

─────────────────────────────────────────────────────────────────────────────

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create
  ~ update in-place

Terraform will perform the following actions:

  # google_cloud_run_service_iam_binding.default will be updated in-place
  ~ resource "google_cloud_run_service_iam_binding" "default" {
        id       = "v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker"
      ~ members  = [
          + "allUsers",
        ]
        # (5 unchanged attributes hidden)
    }

  # google_cloud_run_v2_service.backend will be created
  + resource "google_cloud_run_v2_service" "backend" {
      + conditions              = (known after apply)
      + create_time             = (known after apply)
      + creator                 = (known after apply)
      + delete_time             = (known after apply)
      + effective_annotations   = (known after apply)
      + effective_labels        = (known after apply)
      + etag                    = (known after apply)
      + expire_time             = (known after apply)
      + generation              = (known after apply)
      + id                      = (known after apply)
      + ingress                 = (known after apply)
      + last_modifier           = (known after apply)
      + latest_created_revision = (known after apply)
      + latest_ready_revision   = (known after apply)
      + launch_stage            = (known after apply)
      + location                = "europe-west1"
      + name                    = "hypnobuddy-backend"
      + observed_generation     = (known after apply)
      + project                 = "imi-b4"
      + reconciling             = (known after apply)
      + terminal_condition      = (known after apply)
      + terraform_labels        = (known after apply)
      + traffic_statuses        = (known after apply)
      + uid                     = (known after apply)
      + update_time             = (known after apply)
      + uri                     = (known after apply)

      + template {
          + max_instance_request_concurrency = (known after apply)
          + service_account                  = (known after apply)
          + timeout                          = (known after apply)

          + containers {
              + image = "justingebert/hypnobuddy-backend:latest"

              + env {
                  + name  = "MONGO_URL"
                  + value = (sensitive value)
                }
            }
        }
    }

Plan: 1 to add, 1 to change, 0 to destroy.

─────────────────────────────────────────────────────────────────────────────

Note: You didn't use the -out option to save this plan, so Terraform can't
guarantee to take exactly these actions if you run "terraform apply" now.
```

justingebert commented:
Terraform Plan Output:
```Running plan in Terraform Cloud. Output will stream here. Pressing Ctrl-C
will stop streaming the logs, but will not stop the plan running remotely.

Preparing the remote plan...

To view this run in a browser, visit:
https://app.terraform.io/app/hypnobuddy/hypnobuddy/runs/run-Hw4m7jJLTRH7Qo3H

Waiting for the plan to start...

Terraform v1.7.1
on linux_amd64
Initializing plugins and modules...
data.google_secret_manager_secret_version.mongo_url: Refreshing...
data.google_secret_manager_secret_version.mongo_url: Refresh complete after 1s [id=projects/897359304801/secrets/mongo_url/versions/1]
google_cloud_run_v2_service.backend: Refreshing state... [id=projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend]
google_cloud_run_service_iam_binding.default: Refreshing state... [id=v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker]
google_cloud_run_service_iam_binding.default: Drift detected (update)
google_cloud_run_v2_service.backend: Drift detected (delete)

Note: Objects have changed outside of Terraform

Terraform detected the following changes made outside of Terraform since the
last "terraform apply" which may have affected this plan:

  # google_cloud_run_v2_service.backend has been deleted
  - resource "google_cloud_run_v2_service" "backend" {
        id                      = "projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend"
      - location                = "europe-west1" -> null
      - name                    = "hypnobuddy-backend" -> null
        # (24 unchanged attributes hidden)

        # (2 unchanged blocks hidden)
    }


Unless you have made equivalent changes to your configuration, or ignored the
relevant attributes using ignore_changes, the following plan may include
actions to undo or respond to these changes.

─────────────────────────────────────────────────────────────────────────────

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create
  ~ update in-place

Terraform will perform the following actions:

  # google_cloud_run_service_iam_binding.default will be updated in-place
  ~ resource "google_cloud_run_service_iam_binding" "default" {
        id       = "v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker"
      ~ members  = [
          + "allUsers",
        ]
        # (5 unchanged attributes hidden)
    }

  # google_cloud_run_v2_service.backend will be created
  + resource "google_cloud_run_v2_service" "backend" {
      + conditions              = (known after apply)
      + create_time             = (known after apply)
      + creator                 = (known after apply)
      + delete_time             = (known after apply)
      + effective_annotations   = (known after apply)
      + effective_labels        = (known after apply)
      + etag                    = (known after apply)
      + expire_time             = (known after apply)
      + generation              = (known after apply)
      + id                      = (known after apply)
      + ingress                 = (known after apply)
      + last_modifier           = (known after apply)
      + latest_created_revision = (known after apply)
      + latest_ready_revision   = (known after apply)
      + launch_stage            = (known after apply)
      + location                = "europe-west1"
      + name                    = "hypnobuddy-backend"
      + observed_generation     = (known after apply)
      + project                 = "imi-b4"
      + reconciling             = (known after apply)
      + terminal_condition      = (known after apply)
      + terraform_labels        = (known after apply)
      + traffic_statuses        = (known after apply)
      + uid                     = (known after apply)
      + update_time             = (known after apply)
      + uri                     = (known after apply)

      + template {
          + max_instance_request_concurrency = (known after apply)
          + service_account                  = (known after apply)
          + timeout                          = (known after apply)

          + containers {
              + image = "justingebert/hypnobuddy-backend:latest"

              + env {
                  + name  = "MONGO_URL"
                  + value = (sensitive value)
                }
            }
        }
    }

Plan: 1 to add, 1 to change, 0 to destroy.

─────────────────────────────────────────────────────────────────────────────

Note: You didn't use the -out option to save this plan, so Terraform can't
guarantee to take exactly these actions if you run "terraform apply" now.
```

justingebert commented:
Terraform Plan Output:
```Running plan in Terraform Cloud. Output will stream here. Pressing Ctrl-C
will stop streaming the logs, but will not stop the plan running remotely.

Preparing the remote plan...

To view this run in a browser, visit:
https://app.terraform.io/app/hypnobuddy/hypnobuddy/runs/run-Uakw1bRyBbBSr61p

Waiting for the plan to start...

Terraform v1.7.1
on linux_amd64
Initializing plugins and modules...
data.google_secret_manager_secret_version.mongo_url: Refreshing...
data.google_secret_manager_secret_version.mongo_url: Refresh complete after 0s [id=projects/897359304801/secrets/mongo_url/versions/1]
google_cloud_run_v2_service.backend: Refreshing state... [id=projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend]
google_cloud_run_service_iam_binding.default: Refreshing state... [id=v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker]
google_cloud_run_v2_service.backend: Drift detected (update)

No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration
and found no differences, so no changes are needed.

─────────────────────────────────────────────────────────────────────────────

Note: You didn't use the -out option to save this plan, so Terraform can't
guarantee to take exactly these actions if you run "terraform apply" now.
```

justingebert commented:
Terraform Plan Output:
```Running plan in Terraform Cloud. Output will stream here. Pressing Ctrl-C
will stop streaming the logs, but will not stop the plan running remotely.

Preparing the remote plan...

To view this run in a browser, visit:
https://app.terraform.io/app/hypnobuddy/hypnobuddy/runs/run-izyL6ndFyXB2Evgf

Waiting for the plan to start...

Terraform v1.7.1
on linux_amd64
Initializing plugins and modules...
data.google_secret_manager_secret_version.mongo_url: Refreshing...
data.google_secret_manager_secret_version.mongo_url: Refresh complete after 0s [id=projects/897359304801/secrets/mongo_url/versions/1]
google_cloud_run_v2_service.backend: Refreshing state... [id=projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend]
google_cloud_run_service_iam_binding.default: Refreshing state... [id=v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker]
google_cloud_run_v2_service.backend: Drift detected (update)

No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration
and found no differences, so no changes are needed.

─────────────────────────────────────────────────────────────────────────────

Note: You didn't use the -out option to save this plan, so Terraform can't
guarantee to take exactly these actions if you run "terraform apply" now.
```

justingebert commented:
Terraform Plan Output:
```Running plan in Terraform Cloud. Output will stream here. Pressing Ctrl-C
will stop streaming the logs, but will not stop the plan running remotely.

Preparing the remote plan...

To view this run in a browser, visit:
https://app.terraform.io/app/hypnobuddy/hypnobuddy/runs/run-ubayXqG8CSP3SJJn

Waiting for the plan to start...

Terraform v1.7.1
on linux_amd64
Initializing plugins and modules...
data.google_secret_manager_secret_version.mongo_url: Refreshing...
data.google_secret_manager_secret_version.mongo_url: Refresh complete after 0s [id=projects/897359304801/secrets/mongo_url/versions/1]
google_cloud_run_v2_service.backend: Refreshing state... [id=projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend]
google_cloud_run_service_iam_binding.default: Refreshing state... [id=v1/projects/imi-b4/locations/europe-west1/services/hypnobuddy-backend/roles/run.invoker]
google_cloud_run_v2_service.backend: Drift detected (update)

No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration
and found no differences, so no changes are needed.

─────────────────────────────────────────────────────────────────────────────

Note: You didn't use the -out option to save this plan, so Terraform can't
guarantee to take exactly these actions if you run "terraform apply" now.
```
